### ISSUE TITLE
Document the quantity selector hooks

### ISSUE THREAD

**Issue by tarhi-saad** [woothemes/woocommerce#42601]
> [!CAUTION]
> This issue is blocked by the following issue:
> - [ ] #42047.

The only source of information about the quantity selector hooks is the woocommerce/woocommerce-blocks#5406 PR. It would be more appropriate to include this information in our docs so that it can be easily found.

**Comment by Marc-pi** [comment]
i do confirm the documentation is needed :-)

**Comment by pierorocca** [comment]
Hi @tarhi-saad - thanks for reporting this, and for the detail you gave.

I know this has been open for a very long time, and I appreciate your patience. We're currently going through some older WooCommerce issues, identifying what still needs fixing, and tidying up what's already fixed or out of scope.

It looks like this has already been fixed, or it's no longer happening on trunk.

Here's what I found: The issue asked for documentation of quantity selector hooks that previously only existed in the old WooCommerce Blocks PR #5406. Trunk now documents `woocommerce_add_to_cart_quantity`, `woocommerce_add_to_cart_sold_individually_quantity`, `woocommerce_store_api_product_quantity_limit`, and `woocommerce_store_api_product_quantity_{$value_type}` in the Blocks extensibility hook docs. The global migrated-hooks reference also links to the two add-to-cart quantity entries. This appears fixed from the codebase/docs point of view.

I'm going to close this one. If you're still facing this issue on a current version, please reply with steps to reproduce and we'll happily take another look.

Fix/changelog reference: The requested hook documentation now exists in the Blocks third-party developer hook docs and the migrated-hooks reference links to those entries.
